modules/shared/src/main/scala/io/constellationnetwork/schema/GlobalSnapshotInfo.scala

```diff
@@ -131,5 +136,12 @@ case class GlobalSnapshotInfo(
 }
 
 object GlobalSnapshotInfo {
-  def empty = GlobalSnapshotInfo(SortedMap.empty, SortedMap.empty, SortedMap.empty, SortedMap.empty, SortedMap.empty)
+  def empty = GlobalSnapshotInfo(
+    SortedMap.empty,
+    SortedMap.empty,
+    SortedMap.empty,
+    SortedMap.empty,
+    SortedMap.empty,
+    SortedMap.empty[Address, SortedMap[Address, Balance]].some
+  )
```

Comment:
@gclaramunt we don't want the default values in `GlobalSnapshotInfo` because if someone would have to create global snapshot info, it should provide all values and do not rely on default values. the empty fn is intended to provide all empty values and it's done once only, so there is no DRY rule violated.

Reply:
If you take a look @gclaramunt the empty function is called inside tests only